db/1.8/modules/ROOT/pages/database-connector-connection.adoc

```diff
@@ -0,0 +1,393 @@
+= Configure a Database Connection
+
+Anypoint Connector for Database (Database Connector) can connect to any database for which a JDBC driver is available. The following examples show how to connect to the most popular databases such as Derby, Microsoft SQL Server, MySQL, Oracle, and a generic database. Additionally, for advanced uses cases, there are examples to connect to a global data source, configure a JDBC driver, and configure connection pooling, both in Anypoint Studio and the XML editor.
+
+== Configure Data Source Reference Connection
```

Comment:
I think you need to add "a" here and elsewhere: Configure a Data Source Reference Connection.